Dockerfile.e2e

```diff
@@ -0,0 +1,25 @@
+# Copyright 2019 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Copy the controller-manager into a thin image
+FROM golang:1.13.8 as builder
+WORKDIR /workspace
+COPY bin/manager manager
+RUN chmod 755 manager
+
+FROM gcr.io/distroless/static:latest
+WORKDIR /
+COPY --chown=root:root --from=builder /workspace/manager manager
+USER nobody
+ENTRYPOINT ["/manager"]
```

Comment:
I'd kind of like to avoid having a separate dockerfile for e2e vs release, that way we are more accurately testing what we are planning on releasing.

Reply:
Made it a parameter called "FASTBUILD" to enable using your local go cache.